packages/components/src/theme/types/SistentColor.ts

```diff
@@ -0,0 +1,35 @@
+import * as t from 'io-ts';
+import { TupleCodec } from './Tuple';
+
+const DefaultSistentColorType = t.union([
+  t.literal('dark'),
+  t.literal('gray'),
+  t.literal('red'),
+  t.literal('pink'),
+  t.literal('grape'),
+  t.literal('violet'),
+  t.literal('indigo'),
+  t.literal('blue'),
+  t.literal('cyan'),
+  t.literal('green'),
+  t.literal('lime'),
+  t.literal('yellow'),
+  t.literal('orange'),
+  t.literal('teal'),
```

Comment:
yikes

Reply:
I know. As I said, I need to move some of this to another place, and revert the changes. I wanted to get a PR up to show that I'm working on something